en/docs/learn/exporting-user-info-to-pardot.md

```diff
@@ -0,0 +1,24 @@
+#Exporting WSO2 Identity Server User Information to Pardot
+
+Pardot is a software as a service (SaaS) marketing automation platform by SalesForce offering email automation,
+targeted email campaigns, and lead management for B2B sales and marketing organizations. This topic provides 
+instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the Pardot. 
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create the `users.csv` file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+
+##Try it Out
+
+1. Login to [Pardot](https://pi.pardot.com/).
+2. Navigate to **Admin > Import > Prospects**.
+    ![import-prospect-in-paradot](../assets/img/tutorials/import-prospect-in-paradot.png)
+3. Click **+ Upload File** and choose the created `users.csv` file.
+    ![upload-file-in-paradot](../assets/img/tutorials/upload-file-in-paradot.png)
+4. Click **Next**, after the file get uploaded successfully.
+    ![proceed-after-file-uploading-paradot](../assets/img/tutorials/proceed-after-file-uploading-paradot.png)
+5. Map the column headers to Pardot fields. Pardot suggests some field mappings.
+    ![map-fields-in-pardot](../assets/img/tutorials/map-fields-in-pardot.png)
+6. Complete the **Campaign and Add Tags** and **Action** section according to your needs and confirm user import.
```

Comment:
sections
according to your requirement. 

7.  Confirm user import.

Reply:
Fixed in https://github.com/wso2/docs-is/pull/1765/commits/fc98995b5b535d5cd7b9a5f5b91ca538745a6095